Userland/Libraries/LibGL/GLLights.cpp

```diff
@@ -34,13 +44,13 @@ void glLightModelf(GLenum pname, GLfloat param)
 
 void glLightModelfv(GLenum pname, GLfloat const* params)
 {
-    switch (pname) {
-    case GL_LIGHT_MODEL_AMBIENT:
-        g_gl_context->gl_light_model(pname, params[0], 0.0f, 0.0f, 0.0f);
-        break;
-    default:
-        g_gl_context->gl_light_model(pname, params[0], params[1], params[2], params[3]);
-    }
+    g_gl_context->gl_light_model(pname, params[0], params[1], params[2], params[3]);
```

Comment:
That switch was to prevent the OOB reads :-)

Reply:
But `GL_LIGHT_MODEL_AMBIENT` takes in 4 components?
![image](https://user-images.githubusercontent.com/4768900/150520526-63acfef7-db70-47cb-b949-31371797b60c.png)

Reply:
You are correct! The switch was the wrong way around. We still need to account for `params[1]`, etc. being OOB